app/controllers/openid_connect/authorization_controller.rb

```diff
@@ -70,11 +71,25 @@ def authorization_params
       params.permit(OpenidConnectAuthorizeForm::ATTRS)
     end
 
+    def handle_invalid_logged_out_form
+      return if current_user
+
+      result = @authorize_form.check_submit
+      return if result.success?
+
+      track_authorize_analytics(result)
```

Comment:
Create a new analytics event for this, to separate the validation from the actual authorization? That way, we can differentiate between users who made the request, hence triggering the validation, but did not actually sign in.